packages/components/pages/f-account-info/CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.8.0
+------------------------------
+*December 07, 2021*
+
+### Added
+- `isFormDirty` checks to avoid submitting the form when the data hasn't changed.
```

Comment:
Totally uneducated question on this one; but I noticed when working on validation, there is a this.$v.isDirty or something. Could we use that for the same purpose rather than introducing new properties?

Reply:
Will take a look to see if that will work here 👍

Reply:
@ashleynolan @tommcclean Have looked at this and as far as I can see the `$v.dirty` state is dependant on user interaction, i.e  used to keep fields from showing as invalid, before the user has interacted with them. What we want to do is track the value change on the form rather than just the interaction i.e clicking in and out of the input field (this toggles the $v.dirty value from false > true from what i can see). I think the name `isFormDirty` might be better named to something like `hasFormUpdate` or something to avoid confusion with the vuelidate name. Let me know if I have missed something on the `$v.dirty` docs tho, i could be wrong.